app/ts/simulation/services/EthereumClientService.ts

```diff
@@ -86,6 +86,7 @@ export class EthereumClientService {
 			this.newBlockAttemptCallback(newBlock, this, gotNewBlock)
 			this.cachedBlock = newBlock
 		} catch(error) {
+			console.log(`Failed to get a block`)
```

Comment:
`console.error` maybe more prominent in the logs

Reply:
I don't want to be throwing errors that occur normally and expected at times, as these will show in users "errors" in chrome